chemprop/cli/train.py

```diff
@@ -828,10 +828,7 @@ def train_model(
         try:
             trainer_logger = TensorBoardLogger(model_output_dir, "trainer_logs")
         except ModuleNotFoundError as e:
-            logger.warning(
-                "Unable to import TensorBoardLogger, reverting to CSVLogger (original error: %s).",
-                str(e),
-            )
+            logger.warning(f"Unable to import TensorBoardLogger, reverting to CSVLogger (original error: {str(e)}).")
```

Comment:
```suggestion
            logger.warning(f"Unable to import TensorBoardLogger, reverting to CSVLogger (original error: {e}).")
```
f-strings implicitly call `str`

Reply:
Done in [09855f2](https://github.com/chemprop/chemprop/pull/967/commits/09855f2a5b70cbf34e111a121ed945576ea0d16d)